packages/nuxt/src/core/builder.ts

```diff
@@ -78,3 +78,12 @@ async function bundle (nuxt: Nuxt) {
     throw error
   }
 }
+
+async function loadBuilder (nuxt: Nuxt, builder: string) {
+  for (const root of [nuxt.options.rootDir, import.meta.url]) {
+    const builderPath = await tryResolveModule(builder, root)
```

Comment:
I think we can pass search path directly to the kit util (at least mlly reslver supports it)